libvmaf/src/picture.h

```diff
@@ -21,6 +21,16 @@
 
 #include "libvmaf/picture.h"
 
+typedef struct VmafPicturePrivate {
+    void *cookie;
```

Comment:
The naming of this variable seems weird to me. Naming it cookie makes me assume it will be the same as the cookie i pass to the picture pool inside the config. But actually this cookie is used as picture pool reference. I would like to be able to have a cookie with every private instead of interpreting this cookie as a pool to then get to its config and at the end to its cookie. Furthermore this means the cookie is the same for all pictures in the pool instead of being able to change.